tests/test_wave_01.py

```diff
@@ -2,7 +2,7 @@
 import pytest
 
 
-@pytest.mark.skip(reason="No way to test this feature yet")
+# @pytest.mark.skip(reason="No way to test this feature yet")
```

Comment:
I would recommend getting rid of these `skip` decorators rather than commenting them out. There's no real reason to keep them around now that you've implemented everything so they can be removed to keep the code clean.